packages/sdk/src/modules/did/did.spec.ts

```diff
@@ -18,72 +20,110 @@ describe('Did', () => {
     keyring = new Keyring({ type: 'sr25519' });
     alice = keyring.addFromUri('//Alice');
     did = new Did(api, { pair: alice });
-  });
+  }, 40000);
 
   afterAll(async () => {
     await unsubscribeRuntimeVersion(api);
     await api?.disconnect();
   });
 
   describe('create()', () => {
-    it('should create a new Did', async () => {
-      const name = `test-did-${generateRandomString()}`;
+    // creates, reads, and removes to align with expected
+    it('create single did with no custom fields', async () => {
+      const new_did  = 'did-test-1';
+      await createReadRemove(new_did, did, alice, null);
```

Comment:
updated tests to use functions to test for proper did creation. It creats a did, reads it, and then removes it.